src/sentry/testutils/helpers/datetime.py

```diff
@@ -17,7 +17,7 @@ def before_now(**kwargs):
 
 
 def timestamp_format(datetime):
-    return time.mktime(datetime.utctimetuple()) + datetime.microsecond / 1e6
+    return time.mktime(datetime.utctimetuple()) + datetime.microsecond // 1e6
```

Comment:
Maybe add a comment above?

Reply:
I was completely wrong here, I've updated this with the correct solution